runtime/battery-station/src/lib.rs

```diff
@@ -39,7 +39,7 @@ pub use crate::parachain_params::*;
 pub use crate::parameters::*;
 use alloc::vec;
 use frame_support::{
-    traits::{ConstU16, ConstU32, Contains, EnsureOneOf, EqualPrivilegeOnly, InstanceFilter},
+    traits::{ConstU16, ConstU32, Contains, EitherOfDiverse, EqualPrivilegeOnly, InstanceFilter},
```

Comment:
[nitpick]
I would remove that dependency from `runtime/zeitgeist/src/lib.rs` and `runtime/battery-station/src/lib.rs` and import it directly in `runtime/common/src/lib.rs`.

Reply:
I prefer only changes that are absolutely needed for polkadot upgrade.